ansible/templates/nssp-params-prod.json.j2

```diff
@@ -1,6 +1,7 @@
 {
   "common": {
     "export_dir": "/common/covidcast/receiving/nssp",
+    "backup_dir": "./raw_data_backups",
```

Comment:
where would this location be? I feel like it would be better to be somewhere in /common/

Reply:
This location would be `/home/indicators/runtime/nssp//raw_data_backups`. The location is because of precendents in [chng](https://github.com/cmu-delphi/covidcast-indicators/blob/3450dfc09ff8c4bbc9d89c28ddc78b81f52e2ee6/ansible/templates/changehc-params-prod.json.j2#L7), [hospital admissions](https://github.com/cmu-delphi/covidcast-indicators/blob/3450dfc09ff8c4bbc9d89c28ddc78b81f52e2ee6/ansible/templates/claims_hosp-params-prod.json.j2#L7C1-L7C37), [nchs](https://github.com/cmu-delphi/covidcast-indicators/blob/3450dfc09ff8c4bbc9d89c28ddc78b81f52e2ee6/ansible/templates/nchs_mortality-params-prod.json.j2#L3-L4), [quidel](https://github.com/cmu-delphi/covidcast-indicators/blob/3450dfc09ff8c4bbc9d89c28ddc78b81f52e2ee6/ansible/templates/quidel_covidtest-params-prod.json.j2#L8C3-L8C34). 
Basically, cache files of all sorts are historically stored within the respective dir where the code lives on prod, I assume because we don't want these files modified/disturbed. Meanwhile, common is for stuff that needs interaction with other processes, which is why the normal daily runs all output to common. It's so acquisition can pick them up. The permissions are set up accordingly as well. 
Considering we probably don't want these files modified and just stashed away, I think keeping them where they are is okie.